Engarde_Synthesis/Engarde_Synthesis.csproj

```diff
@@ -11,8 +11,4 @@
     <PackageReference Include="Mutagen.Bethesda.Synthesis" Version="0.18.2" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="FormKeys" />
-  </ItemGroup>
-
```

Comment:
Probably unintended change?

Reply:
VS studio made this change, I'm not exactly sure what this change does. Do you know?

Reply:
I actually don't know, but if it works without these lines, then let them be deleted